deploy/db/scripts/run-postflight-job.k8s.sh

```diff
@@ -1,66 +1,81 @@
 #!/bin/sh
```

Comment:
bash

Reply:
`function` keyword works in sh, and bash isn't available in the postflight container.